ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionReceipt.java

```diff
@@ -180,4 +176,24 @@ private ReceiveResponse getReceiveResponseFromEnclave(
     LOG.trace("Received transaction information from Enclave");
     return enclaveResponse;
   }
+
+  private JsonRpcResponse handleEnclaveException(
+      final JsonRpcRequestContext requestContext, final EnclaveException e) {
+    final JsonRpcError jsonRpcError =
+        JsonRpcEnclaveErrorConverter.convertEnclaveInvalidReason(e.getMessage());
+    switch (jsonRpcError) {
+      case ENCLAVE_PAYLOAD_NOT_FOUND:
+        {
+          return new JsonRpcSuccessResponse(requestContext.getRequest().getId(), null);
+        }
+      case ENCLAVE_KEYS_CANNOT_DECRYPT_PAYLOAD:
+        {
+          LOG.warn(
+              "Unable to decrypt payload with configured privacy node key. Check if your 'privacy-public-key-file' property matches your Orion node public key.");
+        }
+        // fall through
```

Comment:
This is required by errorprone to highlight that we aren't falling to the default branch by mistake.

Reply:
Prudent 👍